src/V3Ast.cpp

```diff
@@ -1098,9 +1102,53 @@ bool AstNode::sameTreeIter(const AstNode* node1p, const AstNode* node2p, bool ig
 void AstNode::checkTreeIter(const AstNode* prevBackp) const VL_MT_STABLE {
     // private: Check a tree and children
     UASSERT_OBJ(prevBackp == this->backp(), this, "Back node inconsistent");
-    switch (this->type()) {
-#include "V3Ast__gen_op_checks.h"
-    default: VL_UNREACHABLE;  // LCOV_EXCL_LINE
+    const VNTypeInfo& typeInfo = *type().typeInfo();
+    for (int i = 1; i <= 4; i++) {
+        AstNode* nodep = nullptr;
+        switch (i) {
+        case 1: nodep = op1p(); break;
+        case 2: nodep = op2p(); break;
+        case 3: nodep = op3p(); break;
+        case 4: nodep = op4p(); break;
+        }
```

Comment:
```suggestion
        case 4: nodep = op4p(); break;
        default: nodep->v3fatalSrc("Bad case");
        }
```

Reply:
The iteration range here is explicitly specified (i=1 to 4). Is it really necessary to fill the default case?

Reply:
Typically I'm defensive for all case statements.

Reply:
Fixed in the new commit